hack/lib/strimzi.bash

```diff
@@ -46,8 +46,8 @@ function install_strimzi_cluster {
           offsets.topic.replication.factor: 3
           transaction.state.log.replication.factor: 3
           transaction.state.log.min.isr: 2
-          inter.broker.protocol.version: "2.8"
-          log.message.format.version: "2.8"
+          inter.broker.protocol.version: "3.0"
```

Comment:
Does that mean that every cluster (4.6+) gets version 3.0? Should not we test against the version we used for the release back then?

Reply:
Ideally we'd test with both, but if we want to release 1.19.1 now, it would be more useful to verify with the latest strimzi (as that would be the one automatically installed on most clusters using strimzi operator)